gen.py

```diff
@@ -72,15 +72,25 @@ class PlotFlags(OutFlags):
     template: FilePath = Field(None, description="Default plot template")
 
 
+# pylint: disable=invalid-name
+class X(BaseModel):
```

Comment:
Let's name it since this is part of only what `x` supports. I don't have good names either to suggest, how about `PlotFileColumnMap`/`PlotFileColumnDict` etc? If you can think of good names, please feel free.